creator-node/src/serviceRegistry.js

```diff
@@ -86,6 +86,10 @@ class ServiceRegistry {
     )
   }
 
+  async initLibs() {
+    this.libs = this.libs || (await this._initAudiusLibs())
+  }
+
```

Comment:
this fixes the issue where we need an initialized libs instance in other node processes - this allows us to create a serviceRegistry instance and initLibs without all the other init steps (and associated latency)